README.md

```diff
@@ -855,6 +856,18 @@ curl -X GET http://cloud.foobar.com/.well-known/apple-app-site-association
 }
 ```
 
+## BI web auth
+
+For some banks integration, BI need something similar to universal link because
+they expect a static domain for fallback but there is a specific domain per Cozy
+instance.
+Compared to universal link, we also need to propagate query parameters provided
```

Comment:
In contrast with universal links, query parameters (provided by BI) are propagated to
the final fallback redirection.